integration_tests/test_aggregator_settlement.py

```diff
@@ -0,0 +1,174 @@
+        pv_client.select_aggregator(self.aggregator_uuid)
+        self.pv_uuid = pv_client.area_uuid
+
+    def _set_first_measurement_forecast(self):
+        self.add_to_batch_commands.set_energy_forecast(
+            asset_uuid=self.load_uuid,
+            energy_forecast_kWh={"2022-01-12T01:00": self.base_forecast_load}) \
+            .set_energy_forecast(
+            asset_uuid=self.pv_uuid,
+            energy_forecast_kWh={"2022-01-12T01:00": self.base_forecast_pv}) \
+            .set_energy_measurement(
+            asset_uuid=self.load_uuid,
+            energy_measurement_kWh={"2022-01-12T00:00": self.base_measurement_load}) \
+            .set_energy_measurement(
+            asset_uuid=self.pv_uuid,
+            energy_measurement_kWh={"2022-01-12T00:00": self.base_measurement_pv})
+
+        self.send_batch_commands()
+
+    @staticmethod
+    def _calculate_random_deviated_energy(energy, off_percentage):
+        delta = energy * off_percentage / 100
+        sign = 1 if random.random() < 0.5 else -1
+        return energy + sign * delta
+
+    @staticmethod
+    def _can_place_forecast_measurements(area_dict):
+        return "area_name" in area_dict and ("forecast-measurement" in area_dict["area_name"])
+
+    @staticmethod
+    def _next_market_slot(market_info):
+        return from_format(market_info["market_slot"], DATE_TIME_FORMAT) \
+            .add(minutes=60).format(DATE_TIME_FORMAT)
+
+    def _manage_forecasts_measurements(self, area_uuid, area_dict,
+                                       forecast_market_slot, current_market_slot):
+        if "load" in area_dict["area_name"]:
+            energy_forecast = self.base_forecast_load
+            energy_measurement = self.base_measurement_load
+        else:
+            energy_forecast = self.base_forecast_pv
+            energy_measurement = self.base_measurement_pv
+
+        self.add_to_batch_commands.set_energy_forecast(asset_uuid=area_uuid,
+                                                       energy_forecast_kWh={
+                                                           forecast_market_slot: energy_forecast
+                                                       }) \
+            .set_energy_measurement(asset_uuid=area_uuid,
+                                    energy_measurement_kWh={
+                                        current_market_slot: energy_measurement
+                                    })
+
+    @staticmethod
+    def _can_place_in_settlement_market(asset_info):
+        return "unsettled_deviation_kWh" in asset_info and (
+                asset_info["unsettled_deviation_kWh"] != 0)
+
+    def on_market_cycle(self, market_info):
+        logging.info("market_info: %s", market_info)
+
+        try:
+            for area_uuid, area_dict in self.latest_grid_tree_flat.items():
+                if not area_dict.get("asset_info"):
+                    continue
+
+                # manage forecast and measurement information
+
```

Comment:
Please go easy on the extra blank lines. "Use blank lines in functions, sparingly, to indicate logical sections." 
Reference in PEP8: https://www.python.org/dev/peps/pep-0008/#blank-lines